### ISSUE TITLE
MemoryStream Set: incorrect get size and write to buffer causes exception (C# generator)

### ISSUE THREAD

**Issue by BlackTr** [chronoxor/FastBinaryEncoding#71]
E.g. we have input stream with Length = 100 and Capacity = 1000.

```
public override void Set(MemoryStream value)
...
uint fbeBytesSize = (uint)value.Length; // here we have 100
...
Write(fbeBytesOffset + 4, value.GetBuffer()); // and trying to write 1000
```

Proposed solution (to c# generator)?
```
 Write(fbeBytesOffset + 4, value.GetBuffer(), 0, fbeBytesSize);
```

**Comment by chronoxor** [comment]
Thanks for reporting! Fixed in 1.11.0.0

**Comment by BlackTr** [comment]
Thank you!
